### ISSUE TITLE
Plug in se queda en modo de pruba

### ISSUE THREAD

**Issue by luisluna85** [conekta/conekta_prestashop#100]
Instale la versión mas reciente del modulo 1.2.0 pero al momento de generar un pedido para pago en efectivo este se aprueba solo como si estuviera en modo sandbox, incluso en la descripción del pedido aparece que el pago fue en modo test, sin embargo tanto el modulo como en conekta todo esta configurado en modo producción

**Comment by angelg4a** [comment]
Lograste solucionar? Tengo el mismo issue

**Comment by luisluna85** [comment]
No, no lo pudimos solucionar, optamos por usar stripe y openpay
